devito/passes/clusters/aliases.py

```diff
@@ -398,22 +508,22 @@ class Alias(object):
     is tracked.
     """
 
-    def __init__(self, alias, aliased=None, distances=None, ghost_offsets=None):
+    def __init__(self, alias, aliased, distances, ghost_offsets=None):
         self.alias = alias
-        self.aliased = aliased or []
-        self.distances = distances or []
+        self.aliased = aliased
+        self.distances = distances
         self.ghost_offsets = ghost_offsets or Stencil()
 
         assert len(self.aliased) == len(self.distances)
 
         # Transposed distances
         self.Tdistances = LabeledVector.transpose(*distances)
 
-    @property
+    @cached_property
     def dimensions(self):
         return tuple(i for i, _ in self.Tdistances)
 
-    @property
+    @cached_property
```

Comment:
So this was here before, but what is an anti-stencil?

Reply:
had to drop this concept in #1171 so you can ignore it